main/py-mock/APKBUILD

```diff
@@ -27,15 +25,37 @@ prepare() {
 }
```

Comment:
Please remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.